### PR TITLE
Fix Promo colors on login screen for native devices

### PR DESCRIPTION
**Pull request by parasharrajat**
### Fixed Issues

$ https://github.com/Expensify/App/issues/6070

### Tests | QA Steps
1. Launch the app
2. Observe the PromoColor image on the login page

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web | Desktop

image

#### Mobile Web

image

#### iPAD

Screenshot 2021-11-03 10:28:29

#### Android Tablet

Screenshot 2021-11-03 09:47:39

### CODE REVIEW

**Comment by parasharrajat** [comment]
Gentle Bump @roryabraham This is ready.

**Review by roryabraham — COMMENTED** [review]
Looks good, just one minor thing...

File: src/components/SVGImage/index.native.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+
```
[the remark this answers is not in the document]
Comment (parasharrajat) [reply src/components/SVGImage/index.native.js]: Done

**Comment by roryabraham** [comment]
Still wonky on iPhone 13 landscape, but I was able to confirm that this PR fixes the issue:

image

It would be nice to have the solid color fill the safeArea on landscape mode, but I think that's out of scope for this.

**Review by roryabraham — APPROVED** [review]
LGTM and tests well (at least, fixes the linked issue) 👍

**Comment by roryabraham** [comment]
Going to merge and test this GH extension feature:

image

**Comment by parasharrajat** [comment]
> It would be nice to have the solid color fill the safeArea on landscape mode, but I think that's out of scope for this.

Yeah, Good suggestion. I am working on another PR related to login page. I can try to add that there if not much effforts.

**Comment by roryabraham** [comment]
Nice, that extension `Wait for successful checks` feature ^ works as long as you keep the tab open

**Pull request merged**